acceptance/tests/resource/service/service_enable_linux.rb

```diff
@@ -119,10 +124,17 @@
     if is_sysV
       rc_symlinks = on(agent, "find /etc/ -name *#{package_name[platform]}", :accept_all_exit_codes => true).stdout
       # sles removes rc.d symlinks
-      if platform != 'sles'
+      if platform != 'sles' && (platform != 'debian' && majrelease != 8)
         (start_runlevels + kill_runlevels).each do |runlevel|
```

Comment:
So is this entire step a skip on sles? If so, it would be clearer if the `platform != sles` conditional were up around the entire step block.